### ISSUE TITLE
Suggestions not displaying for specific queries

### ISSUE THREAD

**Issue by bram1028** [corejavascript/typeahead.js#88]
I have a weird problem where I'm getting no results for certain queries. I've verified the server is returning the correct data, but typeahead is acting like there are no results (shows the notFound template). My data looks like this:

```
[
  {"id":"123","name":"BRADLEY BEECH"},
  {"id":"456","name":"BRADLEY SMITH"},
  {"id":"789","name":"BRADY BLACKBURN"},
  {"id":"234","name":"BRADY FINN"}
]
```

But when I search for ``BRAD``, I get no results. ``BRA``, ``BRADL``, and ``BRADY`` work fine. I have also discovered that ``CARR`` doesn't work while ``CAR`` does. But, other 4 character queries work fine, as well. So, I'm very confused!

Here's my setup:

```
$('#patronLookup [name=query]').typeahead(null, {
    name: 'patron-lookup',
    limit: 10,
    source: new Bloodhound({
        datumTokenizer: Bloodhound.tokenizers.obj.whitespace(['name']),
        queryTokenizer: Bloodhound.tokenizers.whitespace,
        remote: {
            url: '/patron/search?query=%QUERY',
            wildcard: '%QUERY'
        }
    }),
    templates: {
        suggestion(p) {
            return '<div>' + p.name + '</div>'
        },
        notFound: 'No results found!'
    },
    display(p) {
        return p.name
    }
}).on('typeahead:select', (e, o) => {
    location.href = '/patron/' + o.id
});

```

Has anyone else experienced this?

**Comment by bram1028** [comment]
I switched to ``prefetch`` and the suggestions are displayed correctly with ``BRAD`` and ``CARR``. So there's definitely something wrong with ``remote``. I just don't have time to dig into it at the moment.

**Comment by svbergerem** [comment]
@bram1028 Did you try this with the code from the master branch? This looks like #43 which has been fixed with #52 but the fix hasn't been released yet.

**Comment by bram1028** [comment]
@svbergerem That works. Thank you!
